### PR TITLE
Update x11-clipboard to "0.6.1"

### PR DESCRIPTION
**Pull request by xliiv**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -26,5 +26,5 @@ objc_id = "0.1"
 objc-foundation = "0.1"
 
 [target.'cfg(all(unix, not(any(target_os="macos", target_os="android", target_os="ios", target_os="emscripten"))))'.dependencies]
-x11-clipboard = { version = "0.5.1", optional = true }
+x11-clipboard = { version = "0.6", optional = true }
```
Comment (chrisduerr) [inline Cargo.toml:29]:
```suggestion
x11-clipboard = { version = "0.6.1", optional = true }
```

Should **always** specify the minimum semver version actually tested.

**Review by chrisduerr — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (xliiv) [reply Cargo.toml]: sounds reasonable, just updated. thank you

**Comment by chrisduerr** [comment]
> If we want to make update of x11-clipboard possible, we will need to update minimal version of rust to 1.57, for details see #41 (comment)

Updating MSRV is no problem if there's a decent reason to do so. The general guideline is that updating to anything up to the latest version shipped by Ubuntu should be perfectly fine.

**Comment by xliiv** [comment]
Related https://github.com/alacritty/copypasta/pull/43

**Comment by xliiv** [comment]
@chrisduerr please review if you get a chance. thank you :)

**Review by chrisduerr — APPROVED** [review]

**Comment by xliiv** [comment]
1. Can we release a new version to https://crates.io/ ?
2. When could it happen? 
3. Can I help?

Thank you :)

**Pull request merged**